src/libs/analytics.ts

```diff
@@ -1,49 +1,27 @@
 declare global {
   interface Window {
     fathom: any;
```

Comment:
still using fathom? if not, can remove?

Reply:
not being used anymore

Reply:
yep ^ updated in the plausible analytics branch that hasn't been merged in yet